test-packages/test-services/package.json

```diff
@@ -16,6 +16,8 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@sap-cloud-sdk/core": "^1.18.1"
+    "@sap-cloud-sdk/core": "^1.18.1",
```

Comment:
Is this necessary? (Same as above)

Reply:
I tried to remove it and got compilation errors.